app/component/itinerary.scss

```diff
@@ -466,3 +470,31 @@ div.itinerary-container-content {
 .desktop .itinerary-intermediate-stop-name {
   font-size: 15px;
 }
+
+.itinerary-tab .itinerary-timeframe {
+  padding-top: 15px;
+}
+
+.timeframe-nextday {
+  background-color: $gray;
+  line-height: 1;
+  @include font-narrow-medium;
+  color: $white;
+  border-radius: 2px;
+  padding: 4px 5.5px;
```

Comment:
I changed these to use the closest variables instead. Needed to add one for small border radius.